lib/3scale/backend/storage.rb

```diff
@@ -1,301 +1,33 @@
-# used to provide a Redis client based on a configuration object
-require 'uri'
+require '3scale/backend/storage_async'
+require '3scale/backend/storage_sync'
```

Comment:
Just a note: can we design this so opting out of async means no async code gets loaded? I worry about the async monkeypatching stuff, which, while assumed to be inactive, should still not happen when using sync.

Reply:
:+1: